internal/webapi/server.go

```diff
@@ -148,8 +155,22 @@ func (a *Api) removeMachine(rm broadcast.RemoveMachineInfo, r *http.Request, l *
 	return femto.Ok(types.Unit{})
 }
 
-func (a *Api) confirmAdmin(r *http.Request, l *slog.Logger) (*femto.EmptyBodyResponse, error) {
-	return femto.Ok(types.Unit{})
+type UsernameReminder struct {
+	Username string `json:"username"`
+}
+
+func (a *Api) confirmAdmin(auth authentication.Authenticator[APIAuthCredientals], r *http.Request, l *slog.Logger) (*femto.Response[UsernameReminder], error) {
+	// We need to figure out how to get the token out of a request...
+
+	c, err := r.Cookie("token")
```

Comment:
NIT: `"token"` should probably be a `const`, so it's not hardcoded everywhere (doubly so because the frontend doesn't know it, so we should just be able to change it easily)

Reply:
https://github.com/gpuctl/gpuctl/pull/140/commits/19901341f6d288c739e28a5f17a797db042b4d90